core/src/main/web/app/mainapp/components/pluginmanager/pluginmanager-directive.js

```diff
@@ -145,5 +145,19 @@
       }
     };
 
+    $scope.$on(GLOBALS.EVENTS.LANGUAGE_MANAGER_SHOW_SPINNER, function(event, data) {
```

Comment:
I prefer putting the handler on the rootScope and always using $emit instead of $broadcast, unless you really know what you're doing. 

http://toddmotto.com/all-about-angulars-emit-broadcast-on-publish-subscribing/